sql/1.145/maintainer_gh.pl

```diff
@@ -3,30 +3,37 @@
 use FindBin;
 use lib "$FindBin::Dir/../../lib";
 use strict;
+use Try::Tiny;
 use JSON;
 use DDGC;
 
 my $d  = DDGC->new;
 my $ias = $d->rs('InstantAnswer');
+my $is_live = $d->is_live;
 
 while (my $ia = $ias->next) {
     if (my $maintainer = $ia->maintainer) {
-        my %updated_maintainer;
-        my $duckco_user = $d->rs('User')->find({username => $maintainer});
+        try {
+            my $temp_maintainer = from_json($maintainer);
+        } 
+        catch {
+            my %updated_maintainer;
+            my $duckco_user = $d->rs('User')->find({username => $maintainer});
 
-        if ($duckco_user) {
-            %updated_maintainer = ( duckco => $maintainer );
+            if ($duckco_user || (!$is_live)) {
+                %updated_maintainer = ( duckco => $maintainer );
 
-            if ($duckco_user->github_id) {
-                $updated_maintainer{github} = $d->rs('GitHub::User')->find({github_id => $duckco_user->github_id})->login;
+                if ($is_live && $duckco_user->github_id) {
```

Comment:
I don't think we need `$is_live` here.  I have tEStfOUr set up to use my git username and it didn't update correctly.  We should be able to do:
```Perl
if(my $git_id = $duckco_user->github_id){
....
```